### DISCUSSION TITLE
Which Ubuntu release can we install nowadays?

### DISCUSSION THREAD

**Discussion by MeneerJansen** [dnschneid/crouton#4841] [General]
If I run 'crouton -t list' there are no specific Ubuntu versions/releases listed. Just 'xfce', 'lxde-desktop' etc. What Ubuntu release will that install? How can I select a specific Ubuntu release like ver. 22.04 LTS Jammy Jellyfish?

**Comment by DennisLfromGA** [comment]
@MeneerJansen,

`-t` is for *targets*, `-r` is for *releases*

Try: `crouton -r list`

And please take note of the last two lines of the output:
> Releases marked with ! are upstream end-of-life, and should be avoided.
> Releases marked with * are unsupported, but may work with some effort.

*Hope this helps,*
-**DennisLfromGA**

**Reply by MeneerJansen** [reply]
Thanks! I did just that, how do I choose a specific one, e.g. Jammy?

[edit] Got it. It's:
`sudo crouton -t xfce-desktop -r jammy`

I'll pray it works on my antique Chromebook.

**Comment by MeneerJansen** [comment]
Hmmm. I get the following error when trying to install Jammy:
`Extracting .//var/cache/apt/archives/base-files_12ubuntu4_amd64.deb requires the zstdcat command, which is not available`

Trying Trusty (Ubuntu 20.04) now...

**Comment by TomTravis** [comment]
the error is debootstrap need to updated download the current crouton

https://drive.google.com/uc?export=download&id=1bU1Lm8qVqXWqgH5ASZwba006Mz48_AKQ
my cfix will install jammy beware Xorg and xiwi does not work on Jammy cfix has the slowpoke changes installed from the repo crouton

My best fit is focal with mate or KDE you have to install mate or KDE after the chroot is working via 'sudo apt install mate' or [kde-full](https://packages.debian.org/kde-full)

**Reply by MeneerJansen** [reply]
> the error is debootstrap need to updated download the current crouton
> 
> https://drive.google.com/uc?export=download&id=1bU1Lm8qVqXWqgH5ASZwba006Mz48_AKQ my cfix will install jammy beware Xorg and xiwi does not work on Jammy cfix has the slowpoke changes installed from the repo crouton
> 
> My best fit is focal with mate or KDE you have to install mate or KDE after the chroot is working via 'sudo apt install mate' or [kde-full](https://packages.debian.org/kde-full)

Thanks. I installed Debian 11 Bullseye which is reasonably up to date. My previous chroot became unusable due to an update of ChromeOS. I've managed for over 5 years to NOT update ChromeOS but it finally got up to me: it'll update without asking when you restart.

On the plus side: my Chr.B. is so old that another chrome update is impossible, says Google. Chromebooks are super fine hardware, but Google and ChromeOS suck.

**Reply by TomTravis** [reply]
nice that what i use also Bullseye and focal
Bookworm and jammy are being reviewed 
please post your CB ID if you can
with a chrome browser
chrome://system/

**Reply by MeneerJansen** [reply]
Is this what you want to know?

CHROME VERSION |   | 107.0.5304.87

**Reply by TomTravis** [reply]
> Is this what you want to know?
> 
> CHROME VERSION |   | 107.0.5304.87
https://www.ghacks.net/2012/09/04/list-of-chrome-urls-and-their-purpose/
yes the graphic image of the system
![Screenshot 2022-10-25 1 56 00 PM](https://user-images.githubusercontent.com/46386261/198834960-e604a794-2b13-4977-9b3f-32ccba70bdae.png)
 would help Id the hardware

**Reply by MeneerJansen** [reply]
I tried to "downgrade" ChromeOS which didn't solve my problems with trying to use an external USB sound device the way I'm used to with my Chr.B. Installed an old Ubuntu version, didn't help. I now have fresh Debian 11 & XFCE back with no "optimisations" by myself. Don't know yet how to make a screenshot. I'm gonna put the Chr.B. away now. Only thing that I used it for was to produce music with Qtractor and the USB audio device. Since that appears to be impossible nowadays w/ a Chromebook (Crouton or Crositi) I'm probably not gonna do anything with it anymore. Lost interest.

There's another recent topic by me asking what's the matter with Crouton and audio. It looks t ome like Crouton disabled stopping the Cras (ChromeOS Audio Server) service/daemon. I give up. It would take an SystemD expert to explain what's the deal with Cras and Crouton.

**Reply by TomTravis** [reply]
> I tried to "downgrade" ChromeOS which didn't solve my problems with trying to use an external USB sound device the way I'm used to with my Chr.B. Installed an old Ubuntu version, didn't help. I now have fresh Debian 11 & XFCE back with no "optimisations" by myself. Don't know yet how to make a screenshot. I'm gonna put the Chr.B. away now. Only thing that I used it for was to produce music with Qtractor and the USB audio device. Since that appears to be impossible nowadays w/ a Chromebook (Crouton or Crositi) I'm probably not gonna do anything with it anymore. Lost interest.
> 
> There's another recent topic by me asking what's the matter with Crouton and audio. It looks t ome like Crouton disabled stopping the Cras (ChromeOS Audio Server) service/daemon. I give up. It would take an SystemD expert to explain what's the deal with Cras and Crouton.

you cannot down grade you install a specific release if you know what it is via usb 

The model or name of the Chr,B would be helpful
all the software OS release are available via google cloud download cras is running on ChromeOS the Cras on the change root also talk with the Cras on the chromeos best i can tell Cras hardware Pulseaudio is software USB is hardware the drivers for USB audio could be anything
chroot audio is working for me no issue cras is HW device pulseaudio is the software and sound is good

The screen shot is easy via chromeOS select from menu lower right or ctrl+alt + the function button above 6 

https://www.sweetwater.com/store/detail/MPKMini3WT--akai-professional-mpk-mini-mk-iii-limited-editon-white-25-key-keyboard-controller

**Comment by DennisLfromGA** [comment]
Note: You do not have to install the KDE Desktop after you install the chroot, crouton supports two kde targets already:

> kde
        Installs a minimal KDE desktop environment. (Approx. 600MB)
        Requires: x11
        
> kde-desktop
        Installs KDE along with common applications. (Approx. 1000MB)
        Requires: kde

So just add either of them when you install your chroot, or after with the update `-u` option.

*Hope this helps,*
-**DennisLfromGA**

**Reply by TomTravis** [reply]
try it dennis 
The install will sometime fail as dependencies are not complete via debootstrap
apt install updates the  dependencies

**Comment by Dexterhill013** [comment]
I'm on chrome os 132 with chromebrew

**Comment by CroutonIsFun** [comment]
I've found that focal works the best, especially if you want to install qjackctl and qsynth with fluidsynth. To get sound working, unless you can install cras from an older version, you need to buy an inexpensive USB audio adapter, add your username to hwaudio, and then install alsa-utils to set the sound levels.
